controller/workitem.go

```diff
@@ -255,20 +250,32 @@ func (c *WorkitemController) Delete(ctx *app.DeleteWorkitemContext) error {
 	if err != nil {
 		return jsonapi.JSONErrorResponse(ctx, err)
 	}
-	authorized, err := authz.Authorize(ctx, wi.SpaceID.String())
+
+	// This checks if user is space admin or workitem creator or not
```

Comment:
@DhritiShikhar I think the sentence is incomplete. Also, as of now, we do not have the concept of `space admin`. Can you please change it to `space owner` so that we don't have any ambiguity?